internal/lockservice/lockservice.go

```diff
@@ -12,9 +12,10 @@ type LockService interface {
 	// An error is generated if the same isn't possible for any reason,
 	// including releasing locks on non-acquired descriptors.
 	Release(Descriptors) error
-	// CheckAcquire checks whether a lock has been acquired on the given descriptor.
+	// Checkacquire checks whether a lock has been acquired on the given descriptor.
```

Comment:
why has only this function been made lowercase 'a'?

Reply:
Changed